content/cpi.md

```diff
@@ -432,7 +432,7 @@ invoke_signed(
     // Account_infos
     &[token_mint.clone(), user_ata.clone(), mint_auth.clone()],
     // Seeds
-    &[&[b"token_mint", &[mint_bump]]],
+    &[&[b"token_auth", &[mint_auth_bump]]],
```

Comment:
@jamesrp13 can you review this one? I'm 99% sure it's correct but I'm relatively new to anchor to would like a second opinion.